qlib/SqlUtil.qm

```diff
@@ -8545,7 +8548,12 @@ bool b = table.empty();
             *hash oc = opt.(copt.keys());
 
             # process descriptions
-            AbstractDatabase::checkDriverOptions(\desc.columns, drv);
+            #
+            # sqlutil: schema alignment skips column with name "driver" #1684
+            # Do not handle checkDriverOptions for columns hash.
+            # Driver is a potential valid column name. Column sub-hashes
+            # are handled in next section.
+            #AbstractDatabase::checkDriverOptions(\desc.columns, drv);
```

Comment:
I think we should remove this line entirely instead of commenting it out - it should either be there or not and the comments above explain why